pep-0708.rst

```diff
@@ -0,0 +1,849 @@
+
+That is true in that it does improve things, but it has many of the same
+problems as the general ordering idea (though not all of them), but it comes
+along with some of it's own:
+
+- There's no reason to assume that PyPI is the only repository with open
+  registration of names or that it would only be the "default" repository that
+  does this. Projects like Piwheels is an example where users are expected to
+  have a non default repository that also effectively has open registration of
+  names (since it just tracks whatever is registered on PyPI).
+
+
+Rely on repository proxies
+--------------------------
+
+One possible solution is to instead of having the installer have to solve this,
+to instead depend on repository proxies that can intelligently merge multiple
+repositories safely. This could provide a better experience for people with
+complex needs because they can have configuration and features that are
+dedicated to the problem space.
+
+However, that has been rejected because:
+
+- It requires users to opt into using them, unless we also remove the facilities
+  to have more than one repository in installers to force users into using a
+  repository proxy when they need multiple repositories.
+
+  - Removing facilities to have more than one repository configured has been
+    rejected because it would be too disruptive to end users.
+
+- A user may need different outcomes of merging multiple repositories in
+  different contexts, or may need to merge different, mutually exclusive
+  repositories. This means they'll need to actually setup multiple repository
+  proxies for each unique set of options.
+
+- It requires users to maintain infrastructure or it requires adding features in
+  installers to automatically spin up a repository for each invocation.
+
+- It doesn't actually change the requirement to need to have a solution to these
+  problems, it just shifts the responsibility of implementation from installers
+  to some repository proxy, but in either case we still need something that
+  figures out how to merge these disparate namespaces.
+
+- Ultimately, most users do not want to have to stand up a repository proxy just
+  to safely interact with multiple repositories.
+
+
+Rely only on hash checking
+--------------------------
+
+Another possible solution is to rely on hash checking, since with hash checking
+enabled users cannot get an artifact that they didn't expect, it doesn't matter
+if the namespaces are incorrectly merged or not.
+
+This is certainly a solution, unfortunately it also suffers from problems that
+make it unworkable:
+
+- It requires users to opt in to it, so users are still unprotected by default.
+- It requires users to do a bunch of labor to manage their hashes, which is
+  something that most users are unlikely to be willing to do.
+- It is difficult and verbose to get the protection when not using a
+  ``requirements.txt`` file as the source of your dependencies (this affects
```

Comment:
```suggestion
- It is difficult and verbose to get the protection when users are not using a
  ``requirements.txt`` file as the source of their dependencies (this affects
```

Fix conflicting tense, as the former two bullets refer to users in the third person, not the second